src/System.Net.Http/tests/FunctionalTests/CustomContent.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Text;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public class CustomContent : StreamContent
```

Comment:
@pgavlin Good points. I didn't realize that. I'll try to be more `sealed` in my coding life.